### ISSUE TITLE
Can't run test suite

### ISSUE THREAD

**Issue by wakproductions** [Patreon/patreon-ruby#15]
```
patreon-ruby (master)$ rspec

An error occurred while loading ./spec/api_spec.rb.
Failure/Error:
  describe Patreon::API do
    before do
      @api = Patreon::API.new("some token")
    end
  
    describe "Patreon::API#fetch_user" do
      before(:all) do
        @response = File.read(File.expand_path("fixtures/current_user.json", __dir__))
      end

NameError:
  uninitialized constant Patreon::API
# ./spec/api_spec.rb:1:in `<top (required)>'

An error occurred while loading ./spec/util_spec.rb.
Failure/Error:
  describe Patreon::Utils::JSONAPI::URLUtil do
    describe "build_url" do
      it "should build a url" do
        url = Patreon::Utils::JSONAPI::URLUtil.build_url("/")
        assert_equal url, "/"
      end
  
      it "should build a url with a string includes" do
        url = Patreon::Utils::JSONAPI::URLUtil.build_url("/", "include")
        assert_equal url, "/?include=include"

NameError:
  uninitialized constant Patreon::Utils
# ./spec/util_spec.rb:1:in `<top (required)>'
No examples found.
```

**Comment by nperickson** [comment]
@wakproductions - Thanks for reporting this. I've created a bug report for my team to review this.

**Comment by emosesPatreon** [comment]
Hi there.  We use minitest, not rspec.  Until the recently-released 0.5.1, there was also a bug running the minitest suite, but 

    $ rake test

Should now execute our tests properly.  Please update to 0.5.1, and let me know if you have any more problems.
